src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -273,7 +269,7 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, EvalVisitorCo
                 {
                     try
                     {
-                        result = await ptr(this, context.IncrementStackDepthCounter(childContext), node.IRContext, args);
+                        result = await ptr(this, context.IncrementStackDepthCounter(childContext), node.IRContext, args).ConfigureAwait(false);

```

Comment:
Do we need the changes in this file?
Or were they temporary to help discover which tests could be moved?